python/examples/test_addptr.py

```diff
@@ -0,0 +1,46 @@
+import torch
+
+import triton
+from triton.backends.triton_shared.driver import CPUDriver
+import triton.language as tl
+
+triton.runtime.driver.set_active(CPUDriver())
+
+
+@triton.jit
+def addptr(in0, out0):
+    for i in range(0, 10, 2):
```

Comment:
do you want a simple example with block_ptr?

Reply:
added a new blockptr test